src/spa/account/PageActivate.tsx

```diff
@@ -16,11 +16,32 @@ export const PageActivate = () => {
   } = useActivateAccount();
 
   const [searchParams] = useSearchParams();
-
+  const toast = useToast();
+  const navigate = useNavigate();
   useEffect(() => {
     activateAccount({ key: searchParams.get('key') ?? 'KEY_NOT_DEFINED' });
   }, [activateAccount, searchParams]);
 
+  if (isError) {
+    navigate('/');
+    toast({
+      title: t('account:activate.feedbacks.activationError.title'),
+      status: 'error',
+      duration: 9000,
+      isClosable: true,
+      position: 'top-right',
+    });
+  }
+  if (isSuccess) {
+    navigate('/');
+    toast({
```

Comment:
Please use already defined toast function to get the same UI and behavior

https://github.com/BearStudio/start-ui-web/blob/eb62ec5d0e20d51e17280af690cdfd23a654dae3/src/components/Toast/index.ts#L23

Reply:
fixed